source/gui/settingsDialogs.py

```diff
@@ -4363,8 +4363,22 @@ def makeSettings(self, settingsSizer):
 			wx.CheckBox(self.followCursorGroupBox, label=readByParagraphText),
 		)
 		self.bindHelpEvent("BrailleSettingsReadByParagraph", self.readByParagraphCheckBox)
+		self.readByParagraphCheckBox.Bind(wx.EVT_CHECKBOX, self.onReadByParagraphChange)
 		self.readByParagraphCheckBox.Value = config.conf["braille"]["readByParagraph"]
 
+		self.showParagraphStartCombo: nvdaControls.FeatureFlagCombo = followCursorGroupHelper.addLabeledControl(
+			labelText=_(
+				# Translators: This is a label for a combo-box in the Braille settings panel to select if start of paragraphs will be reported in braille.
+				"Show para&graph start",
+			),
+			wxCtrlClass=nvdaControls.FeatureFlagCombo,
+			keyPath=["braille", "showParagraphStart"],
+			conf=config.conf,
+		)
+		self.bindHelpEvent("BrailleSettingsShowParagraphStart", self.showParagraphStartCombo)
```

Comment:
The most recent usage is to avoid using settings panel name in the anchor name, in case options are moved in the future. For example, with the increasing number of braille options, we may imagine that we will have 2 separate (and more specific) braille settings panels.

```suggestion
		self.bindHelpEvent("BrailleShowParagraphStart", self.showParagraphStartCombo)
```